libs/common/src/api/mykomap-openapi.json

```diff
@@ -868,6 +868,9 @@
                       "properties": {
                         "directory_panel_field": {
                           "type": "string"
+                        },
+                        "marker_property_name": {
+                          "type": "string"
```

Comment:
I find commits to this file, being generated automatically, want to be in their own commits - like those to `package-lock.json`. Otherwise later on they have a tendency to tangle up with other logical changes when merging/rebasing etc.